packages/shared/src/components/DevCardPlaceholder.tsx

```diff
@@ -1,21 +1,33 @@
 import React, { ReactElement } from 'react';
 
+const getColorPrefix = (rank: number, isLocked?: boolean) => {
+  if (isLocked) {
+    return '--theme-rank-1-color';
+  }
+
+  return rank > 0 ? `--theme-rank-${rank}-color` : '--theme-rank-5-color';
```

Comment:
@sshanzel is the default 5 rank ok here?

Reply:
Not sure which one to use, though for the areas that used to be using the default is now utilizing what's the user's actual rank.

Reply:
Let's use this for now 🙌